sorting_algorithms.py

```diff
@@ -0,0 +1,52 @@
+import random
+
+
+def selection_sort(xs):
+    for i in range(len(xs)):
+        min = i
+
+        for j in range(i + 1, len(xs)):
+            if xs[min] > xs[j]:
+                min = j
+
+        xs[i], xs[min] = xs[min], xs[i]
+    return xs
+
```

Comment:
trzymaj jedną linię odstępu między funkcjami

Reply:
Nie jest to chyba błąd w tym przypadku, PEP8 mówi:
_Surround top-level function and class definitions with two blank lines.
Method definitions inside a class are surrounded by a single blank line._